app/views/feedbacks/report.xml.erb

```diff
@@ -2,7 +2,7 @@
   <row no="1">
     <fl val="Contact Name"><%= cdata_section(@report.username.present? ? @report.username : "Anonymous user") %></fl>
     <fl val="Email"><%= cdata_section(@report.email.present? ? @report.email : "do-not-reply@archiveofourown.org") %></fl>
-    <fl val="Subject"><%= cdata_section(@report.title.present? ? @report.title.html_safe : "No Title") %></fl>
+    <fl val="Subject"><%= cdata_section(@report.title.present? ? "[#{ArchiveConfig.APP_SHORT_NAME}] Support - #{@report.title.html_safe}": "[#{ArchiveConfig.APP_SHORT_NAME}] Support - No Title") %></fl>
```

Comment:
A little late, but since we might have to poke this anyway, I wanted to mention there's a space missing before the : here: `"[#{ArchiveConfig.APP_SHORT_NAME}] Support - #{@report.title.html_safe}": "[#{ArchiveConfig.APP_SHORT_NAME}]`